.gitignore

```diff
@@ -18,5 +18,5 @@
 /spec/dummy/.controlplane/controlplane*-tmp-*.yml
 
 # Generated configs
-terraform/
-.controlplane/
+/terraform/
```

Comment:
We need to ignore .controlplane folder on root level, otherwise folder spec/dummy/.controlplane is also ignored